### PR TITLE
Small fixes to the Staking tutorial

### PR DESCRIPTION
**Pull request by fadeev**
Related: https://github.com/zeta-chain/example-contracts/pull/76

### CODE REVIEW

**Review by andresaiello — APPROVED** [review]

**Comment by lumtis** [comment]
@fadeev can you wait before merge? I have some issues testing the staking contract on mocknet maybe it's related the changes introduced

**Comment by fadeev** [comment]
@lumtis should I go forward with the changes?

**Comment by lumtis** [comment]
> @lumtis should I go forward with the changes?

I've been a bit busy recently with audit preparation. Can we keep it on hold until early next week?

**Pull request closed without merging**
